class/class3.1/src/Helpers.cpp

```diff
@@ -0,0 +1,16 @@
+#include "Helpers.hpp"
+#include <algorithm> 
+
+double dummyFunc(double x) {
+    return x > 0 ? x : 0;
+}
+
+std::string repeatString(const std::vector<int>& nums, const std::string& str) {
+    if (nums.empty()) return "";
+    int max_num = *std::max_element(nums.begin(), nums.end());
+    std::string result;
+    for (int i = 0; i < max_num; ++i) {
```

Comment:
With std::ranges it can be even simpler
```
auto repeatCount = vec.empty() ? 0 : std::ranges::max(vec);
return repeatCount > 0 ?
            ? std::views::repeat(str, repeatCount) | std::views::join | std::ranges::to<std::string>()
            : "";
```